source/common/runtime/runtime_features.cc

```diff
@@ -70,9 +70,10 @@ constexpr const char* runtime_features[] = {
     "envoy.reloadable_features.hash_multiple_header_values",
     "envoy.reloadable_features.health_check.graceful_goaway_handling",
     "envoy.reloadable_features.health_check.immediate_failure_exclude_from_cluster",
+    "envoy.reloadable_features.http2_consume_stream_refused_errors",
+    "envoy.reloadable_features.http2_skip_encoding_empty_trailers",
```

Comment:
Could you remove `http2_skip_encoding_empty_trailers` from `UNSORTED_FLAGS` in `tools/code_format/check_format.py`